### PR TITLE
Removed version testing for xtrabackups 8.0.27

### PR DESCRIPTION
**Pull request by kathia-barahona**
Checks were raising errors for this version and we currently don't test it anyways.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ repos:
         args: [--ignore-missing-imports]
         additional_dependencies: [types-PyMySQL==1.0.19, types-requests==2.27.30]
 -   repo: https://github.com/pycqa/isort
-    rev: 5.10.1
+    rev: 5.12.0
```
Comment (kathia-barahona) [inline .pre-commit-config.yaml:28]: Seems there was an incompatibility with poetry, was raising errors. https://github.com/PyCQA/isort/releases/tag/5.11.5

 Bumped it to latest version.

Comment (aris-aiven) [reply .pre-commit-config.yaml:28]: It seems we had the same reaction at the same time. Could you look at #139 which also updates the other checks?

Comment (kathia-barahona) [reply .pre-commit-config.yaml:28]: yeeeeh was going to say the same haha

Comment (kathia-barahona) [reply .pre-commit-config.yaml:28]: will revert my changes from here

Comment (aris-aiven) [reply .pre-commit-config.yaml:28]: I may just merge this one to unblock main, and follow up with the other updates

**Review by aris-aiven — APPROVED** [review]

**Pull request merged**